scalafix-testkit/src/main/scala/scalafix/testkit/TestkitPath.scala

```diff
@@ -25,14 +25,6 @@ final class TestkitPath(
     val testPath: RelativePath,
     val semanticdbPath: RelativePath
 ) {
-  override def toString: String = {
-    val map = Map(
-      "input" -> input,
-      "testPath" -> testPath,
-      "semanticdbPath" -> semanticdbPath
-    )
-    pprint.PPrinter.BlackWhite.tokenize(map).mkString
```

Comment:
Only `pprint.TPrint` has been shaded into metaconfig. I felt like other usages of `pprint` here and below were not worth adding an explicit compile-time dependency to `pprint`